### ISSUE TITLE
Tracking issue for modules we've noticed are ported to Oct 2019 - Oct 2020

### ISSUE THREAD

**Issue by mhdawson** [nodejs/abi-stable-node#387]
Issue for 2018-2019 - (https://github.com/nodejs/abi-stable-node/issues/346)

**Comment by gabrielschulhof** [comment]
@vweevers has created a list of native addons that are (not yet) ported to N-API. The list exists for a different reason, but I believe it is relevant to us because it gives us targets for which to evangelize and/or PR the move to N-API.

https://github.com/vweevers/about-native-modules

**Comment by vweevers** [comment]
@gabrielschulhof I can extend the list if there's interest.

**Comment by NickNaso** [comment]
@vweevers Yes please because we can use in our evangelism activities. We already have a list of native add-on here https://docs.google.com/spreadsheets/d/1HopFSLMs5gj3G6_eBEZM2ET7oXKdokAF7VU2XZnzYGk

**Comment by gabrielschulhof** [comment]
@NickNaso I added sodium-native to the top of that list.

**Comment by vweevers** [comment]
WIP: https://github.com/vweevers/about-native-modules#data

**Comment by vweevers** [comment]
In case y'all missed it: I also committed the data as JSON, to easily filter it. For example, to get the top 100 non-napi addons:

```js
const addons = require('about-native-modules/data.json')
const top = addons.filter(addon => !addon.napi).slice(0, 100)

console.log(top.map(addon => addon.name))
```

**Comment by andreek** [comment]
Module websocket is considering to use napi versions of bufferutil and utf-8-validate: https://github.com/theturtle32/WebSocket-Node/pull/302#issuecomment-562740142

**Comment by lovell** [comment]
Hi everyone, thanks for the effort you've made to get N-API stable and provide `node-addon-api` to ease working with it. You'll be pleased to learn that the latest version of [sharp](https://www.npmjs.com/package/sharp) has now migrated to it from `nan`.

According to the data in the linked https://github.com/vweevers/about-native-modules repo, it looks like `sharp` is now the most-downloaded native module to be using N-API!

**Comment by gabrielschulhof** [comment]
I also noticed that simdjson was using node-addon-api.

**Comment by NickNaso** [comment]
I want report this new **Rust** binding to **N-API**: https://github.com/napi-rs/napi-rs

**Comment by NickNaso** [comment]
Implementation of webassembly code interfaces based on N-API https://github.com/yjjnls/napi-emscripten

**Comment by gabrielschulhof** [comment]
@jschlight mentioned that sqlite3 v5.0.0 is running on N-API: https://github.com/mapbox/node-sqlite3/releases/tag/v5.0.0

**Comment by NickNaso** [comment]
I contributed to port this module https://github.com/lovell/highwayhash the next release will be a N-API add-on.

**Comment by andreek** [comment]
> Module websocket is considering to use napi versions of bufferutil and utf-8-validate: https://github.com/theturtle32/WebSocket-Node/pull/302#issuecomment-562740142

v1.0.32 has been released and is using N-API modules from ws project.

**Comment by jschlight** [comment]
I've updated Matteo's [`native-hdr-histogram`](https://github.com/mcollina/native-hdr-histogram) binding to N-API 3 using my automated binding tool. Now published at v1.0.0, the project is completely modernized using [`prebuildify `](https://github.com/prebuild/prebuildify) and GitHub Actions for testing, prebuild, and publishing.
